### PR TITLE
Refacto voucher part with boostrap components

### PR DESCRIPTION
**Pull request by NeOMakinG**
## How does it looks?
image

@Hlavtox @mparvazi if you can review, would be appreciated!

### CODE REVIEW

**Comment by mparvazi** [comment]
Need to add some help comment:
`If you have a voucher code paste here or click on voucher code in the voucher list.`

**Comment by NeOMakinG** [comment]
> Need to add some help comment: `If you have a voucher code paste here or click on voucher code in the voucher list.`
> 
> Some customers have problem with this UX.

Can we talk about this on https://github.com/PrestaShop/theme-refacto/issues/51 ?

I think even your help comment is not clear, we should talk about it

**Comment by NeOMakinG** [comment]
@SharakPL @mparvazi I've addressed your feedbacks, it should be good now

image

wdyt?

**Comment by SharakPL** [comment]
Great! Thanks

File: templates/checkout/_partials/cart-voucher.tpl [hunk 1]
```diff
@@ -34,7 +34,7 @@
               <li class="cart-voucher__item row">
                 <span class="cart-voucher__name col">{$voucher.name}</span>
                 <div class="d-flex align-items-center justify-content-end col">
-                  <span class="me-2">{$voucher.reduction_formatted}</span>
+                  <span class="me-2 fw-bold">{$voucher.reduction_formatted}</span>
```
Comment (Hlavtox) [inline templates/checkout/_partials/cart-voucher.tpl:37]: @NeOMakinG Maybe ms-2 on the other element? If it's not deletable, there will be space.

File: templates/checkout/_partials/cart-voucher.tpl [hunk 2]
```diff
@@ -34,7 +34,7 @@
               <li class="cart-voucher__item row">
                 <span class="cart-voucher__name col">{$voucher.name}</span>
                 <div class="d-flex align-items-center justify-content-end col">
```
Comment (Hlavtox) [inline templates/checkout/_partials/cart-voucher.tpl:36]: col without row?

File: templates/checkout/_partials/cart-voucher.tpl [hunk 3]
```diff
@@ -79,11 +79,11 @@
         <p class="fw-bold fs-6">
           {l s='Take advantage of our exclusive offers:' d='Shop.Theme.Actions'}
         </p>
-        <ul class="js-discount card-voucher__list">
+        <ul class="js-discount cart-voucher__list">
```
Comment (Hlavtox) [inline templates/checkout/_partials/cart-voucher.tpl:82]: Do we want the same selector here? What about cart-voucher__offers?

File: templates/checkout/_partials/cart-voucher.tpl [hunk 4]
```diff
@@ -79,11 +79,11 @@
...
           {foreach from=$cart.discounts item=discount}
             <li class="cart-voucher__code">
               <span class="label">
```
Comment (Hlavtox) [inline templates/checkout/_partials/cart-voucher.tpl:85]: Extra span

File: templates/checkout/_partials/cart-voucher.tpl [hunk 5]
```diff
@@ -34,7 +34,7 @@
               <li class="cart-voucher__item row">
                 <span class="cart-voucher__name col">{$voucher.name}</span>
                 <div class="d-flex align-items-center justify-content-end col">
-                  <span class="me-2">{$voucher.reduction_formatted}</span>
+                  <span class="me-2 fw-bold">{$voucher.reduction_formatted}</span>
                     {if isset($voucher.code) && $voucher.code !== ''}
```
Comment (Hlavtox) [inline templates/checkout/_partials/cart-voucher.tpl:38]: We should change this in the core. There should be a property `deletable` or something.

**Comment by Hlavtox** [comment]
Some small comments, otherwise rock and roll 🚀

I agree with @SharakPL and @mparvazi - everything must be made for dumbest people you can imagine.

File: templates/checkout/_partials/cart-voucher.tpl (hunk 5 above)
Comment (NeOMakinG) [reply templates/checkout/_partials/cart-voucher.tpl:38]: Yeah but not related to hummingbird

**Comment by NeOMakinG** [comment]
@Hlavtox Addressed all of your feedbacks

Looks mergeable now!

**Comment by NeOMakinG** [comment]
Thanks @Hlavtox @SharakPL @mparvazi for the review, if you find something else later, don't hesitate to contribute or open an issue ❤️

**Pull request merged**